app/src/main/res/layout/fragment_media_detail.xml

```diff
@@ -40,257 +39,273 @@
 
             <!-- Placeholder. Height gets set at runtime based on container size; the initial value is a hack to keep
                  the detail info offscreen until it's placed properly. May be a better way to do this. -->
-            <com.facebook.drawee.view.SimpleDraweeView
-                android:id="@+id/mediaDetailImageView"
+
+            <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="@dimen/dimen_250"
-                app:actualImageScaleType="none" />
+                android:orientation="vertical"
+                android:background="@android:color/transparent"
+                android:id="@+id/mediaDetailImageViewSpacer"
+                />
 
             <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:background="?attr/fragmentCategorisationBackground"
-                android:orientation="vertical"
-                android:padding="@dimen/standard_gap">
+                android:background="?attr/mainBackground"
+                android:orientation="vertical">
 
                 <LinearLayout
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:background="?attr/subBackground"
-                    android:orientation="vertical"
-                    android:padding="@dimen/standard_gap">
+                    android:weightSum="100"
+                    android:background="@color/primaryDarkColor"
```

Comment:
This should be overdrawing the region? The surface you are on is already `android:background="?attr/mainBackground"` is that not correct? Why use `primaryDarkColor`?

Reply:
@macgills actually the `?attr/mainBackground` represents white and dark grey colors in light and dark mode respectively, but as per the designs the background for the title section should be dark blue referenced by `@color/primaryDarkColor`

Reply:
Yes sorry, only looking at the git diff it is hard to evaluate the xml in my head